### PR TITLE
Add min limit when add/remove booster

### PR DESCRIPTION
**Pull request by nz-neooo**

### CODE REVIEW

File: contracts/RewardsBooster.sol [hunk 1]
```diff
@@ -298,6 +299,8 @@ contract RewardsBooster is Initializable, OwnableUpgradeable, IRewardsBooster, S
         deploymentPool.accRewardsPerBooster = accRewardsPerBooster;
         totalBoosterPoint += _amount;
 
+        require(deploymentPool.boosterPoint >= minimumDeploymentBooster, 'RB015');
```
Comment (ianhe8x) [inline contracts/RewardsBooster.sol]: I thought you should check `deploymentPool.accountBooster[_account]`

Comment (ianhe8x) [reply contracts/RewardsBooster.sol]: can you add a test to cover this scenaio? when there is boost bigger than min for a deployment and a user tries to add boost smaller than min, it should fail.

**Review by ianhe8x — CHANGES_REQUESTED** [review]

**Comment by ianhe8x** [comment]
@nz-neooo what do you think of `can not get booster reward if boosted token less than minimum`, since now it is impossible except for existing boosters. Should we remove this amount check in the contract?

**Comment by nz-neooo** [comment]
Now it's limited to the total deployment booster. If we want to apply "minimum", limited to the account will be simple and clear.

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**